src/test/javascript/spec/component/exam/manage/exam-management.service.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { fakeAsync, TestBed, tick } from '@angular/core/testing';
+import { fakeAsync, TestBed, tick, inject } from '@angular/core/testing';
```

Comment:
`'inject' is defined but never used`

Reply:
I had the same problem, could not create new exercise group for newly created exam. And the other buttons (add/import X exercise) disappeared for the exams I created before.